code/game/objects/structures/ai_core.dm

```diff
@@ -243,6 +243,13 @@
 						else
 							A = new /mob/living/silicon/ai(loc, laws, brain.brainmob)
 
+						if(get_overmap()?.faction == "syndicate") //NSV13 start - Syndicate ship radio & traitor setting
+							A.set_zeroth_law("Accomplish the Syndicate's objectives at all costs.", "Accomplish your Syndicate's objectives at all costs.")
```

Comment:
SyndieOS lawset (or an adaptation of it akin to NSV crewsimov) instead? Think it may be neat to have neither side's AI immune to law change based interference. Although this is more of a thought and not a required change.

Reply:
Yeah right now this just makes it law 0 be syndicate and the rest is NSV default (or whatever law board was put in manually when building)

Reply:
Maybe it could be fun to have the laws/allegiance change with the IFF, just a thought for later though.